dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -406,11 +447,21 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
 
     winrt::Windows::Foundation::IAsyncOperation<winrt::Microsoft::Windows::AppNotifications::AppNotificationProgressResult> AppNotificationManager::UpdateAsync(winrt::Microsoft::Windows::AppNotifications::AppNotificationProgressData const data, hstring const tag)
     {
+        if (!IsSupported())
```

Comment:
Is this needed here? I believe the API call below already gates. So ....

Reply:
This check was just to be thorough, but it is redundant. Will fix next iteration.